More documentation (#211)
yiisoft/yii-debug: README.md

Install both [`yiisoft/yii-debug-api`](https://github.com/yiisoft/yii-debug-api) and [`yiisoft/yii-dev-panel`](https://github.com/yiisoft/yii-dev-panel)
to be able to interact with collected data through UI.

## Logging

Specify the filesystem path where collected data will be stored if you use `FileStorage` by adding the lines into the configuration:

```php
return [
    'yiisoft/yii-debug' => [
        // It's default path to store collected payload
        // @runtime = @root/runtime
        'path' => '@runtime/debug',
    ],
    // ...
];
```

## Filtering

> Disabling debugging may help you to debug in production or not to flood to the debug storage with payload which never be used for debug purpose.
> 
> You can specify which routes should not trigger the Debug extension by adding the ones into the configuration:

Reviewer: ```suggestion
Disabling debugging for certain requests or console runs may help you to debug in production or not to flood the debug storage with unuseful payloads.

You can specify which routes should not trigger the Debug extension by adding the ones into the configuration:
```

```php
return [
    'yiisoft/yii-debug' => [
        'ignoredRequests' => [
            '/assets/**',
        ],
    ],
    // ...
];
```

> It uses regular expressions under the hood, but looks simpler for users. See (`\Yiisoft\Strings\WildcardPattern`)[https://github.com/yiisoft/strings#wildcardpattern-usage] for more details.

Reviewer: ```suggestion
See (`\Yiisoft\Strings\WildcardPattern`)[https://github.com/yiisoft/strings#wildcardpattern-usage] for more details about the pattern syntax.
```

No need to describe implementation internals.

> In order to disable debugging console commands you can also specify them into another directive `ignoredCommands`.

Reviewer: ```suggestion
In order to disable debugging certain console commands you can also specify them via `ignoredCommands`.
```

    }
```

## ServiceCollector

> ServiceCollector is a collector that listen all tracked services and collect its arguments, results and errors.

Reviewer: ```suggestion
ServiceCollector is a collector that listens all tracked services and collects its arguments, results and errors.
```

By default, the debug extension has enabled [`\Yiisoft\Yii\Debug\Collector\ServiceCollector`](./src/Collector/ServiceCollector.php) with the following settings:
1. Log arguments
2. Log results
3. Log errors

> It may worse the application performance in development what why we recommend to disable this mechanism in production.

Reviewer: ```suggestion
It may degrade the application performance so it may be a good idea to disable it in production.
```

> You may control the things are being logged by specifying the settings in the configuration:

Reviewer: ```suggestion
You may control what is logged by specifying the settings in the configuration:
```
